xbmc/windowing/osx/OpenGL/WindowControllerMacOS.mm

```diff
@@ -162,6 +163,12 @@ - (void)windowDidChangeScreen:(NSNotification*)notification
   winSystem->WindowChangedScreen();
 }
 
+- (void)windowDidChangeBackingProperties:(NSNotification*)notification
+{
+  // if the backing scale change we need to force resize the window
```

Comment:
note: this will probably need a bit of work when we start working towards supporting HDR/EDR. We should resize only if the backing scale change (whereas this event can be triggered if the colourspace changes). I'll add a TODO for now since we don't have support for colorspace right now.